packages/design-tokens/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@availity/design-tokens",
-  "version": "0.5.0",
+  "version": "0.4.0",
```

Comment:
I'm assuming you meant for the version number to decrease but calling attention jic.

Reply:
Yup! I want the version to be bumped to 0.5.0 so have to start at 0.4.0